app/src/main/res/mipmap-anydpi-v26/picover_logo_round.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do we need round icon if our app supports api 26 and higher ?

Reply:
I saw the option to exclude legacy images and I deleted it